HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -963,625 +956,624 @@ def self.set_unavailable_period_values(out, day_schedule, begin_hour, end_hour,
     end
   end
 
-  # TODO
+  # Default occupants weekday fractions.
```

Comment:
What do you think about moving all this data out of ruby? I.e., Create data/default_schedules.csv (or tsv?), which has 24 weekday + 24 weekend + 12 monthly values for each end use.

Reply:
I like this idea. Can you recall if we had thought of this before, but ultimately decided to go the ruby route?

Reply:
Hmm, I don't recall us discussing it before. I think it makes sense to move towards separating data from code where appropriate. Any other data that would make sense for us to do similar?

If you go this route, make sure to only load the CSV once upfront in `hpxml_defaults.rb` (I assume that's where they all get used), not once for each lookup.